expressEngine/user/service/user.js

```diff
@@ -31,6 +31,19 @@ class Service extends BaseService {
       return this.error(err);
     }
   }
+  deleteUser = async (id) => {
+    try {
+      const isExist = await DBService.findOne(User, {_id: id});
```

Comment:
Create a variable for {_id: id}, so that we need to change it only once if required in the future.

Reply:
Unnecessarily using two queries for deletion can be done using only one:  findOneAndDelete().
Please do so.